aws_lambda_powertools/event_handler/api_gateway.py

```diff
@@ -208,7 +214,7 @@ def __init__(
         self,
         status_code: int,
         content_type: Optional[str] = None,
-        body: Union[str, bytes, None] = None,
+        body: Union[Any, None] = None,
```

Comment:
When validation is on, the response can be anything, ranging from strings to data classes to objects. This is why we need the change here, so mypy is happy.

Reply:
```suggestion
        body: Any = None,
```

Let's use `Any`, and create a maintenance issue to make `_to_response()` improve error handling by catching a `TypeError`.

Reply:
Done!